### PR TITLE
Fixed-> CPP14 grammar:parse error when writing a string in multiple l…

### PR DESCRIPTION
**Pull request by rafeeque1**
…ines #1381

Fixed ,Now Its Parsed .

### CODE REVIEW

File: cpp/CPP14.g4 [hunk 1]
```diff
@@ -32,7 +32,7 @@ translationunit
 
 
 primaryexpression
-   : literal
+   : literal (literal)*
```
Comment (KvanTTT) [inline cpp/CPP14.g4:35]: Please remove redundant parentheses and use `+` operator:

```suggestion
   : literal+
```

File: cpp/examples/test.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#include <iostream>
```
Comment (KvanTTT) [inline cpp/examples/test.txt:1]: Use `.cpp` extension for example files:

```
cpp/examples/test.cpp
```

File: cpp/CPP14.g4 (hunk 1 above)
Comment (rafeeque1) [reply cpp/CPP14.g4:35]: Thank you KvanTTT

Comment (rafeeque1) [reply cpp/CPP14.g4:35]: Problems with CPP14.g4 #1577  duplicate issues.

Comment (KvanTTT) [reply cpp/CPP14.g4:35]: What do you mean? I guess `literal (literal)*` and `literal+` are equivalent forms.

**Comment by teverett** [comment]
@rafeeque1 thanks

**Pull request merged**